.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```

Comment:
change according to previous comment

Reply:
I have added .idea to gitignore file and removed the cached version using the git command.

Reply:
@alvinMemphis review